### PR TITLE
Update URL_Filters Demo

### PR DESCRIPTION
**Pull request by ghsable**
##### Description of change

* README.md: Update URL(Filters Demo)
  * `http://pixijs.io/pixi-filters/tools/demo/` -> `https://filters.pixijs.download/v3.2.0/demo/index.html`

##### Pre-Merge Checklist

- [x] Tests and/or benchmarks are included
- [x] Documentation is changed or added
- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by bigtimebuddy** [comment]
Should actually be https://pixijs.io/filters/tools/demo

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ _Note: `5.1.3` can be replaced by any [released](https://github.com/pixijs/pixi.
 
 ### Demos ###
 
-- [Filters Demo](http://pixijs.io/pixi-filters/tools/demo/)
+- [Filters Demo](https://filters.pixijs.download/v3.2.0/demo/index.html)
```
Comment (bigtimebuddy) [inline README.md:77]:
```suggestion
- [Filters Demo](https://pixijs.io/filters/tools/demo/)
```

**Review by bigtimebuddy — APPROVED** [review]

**Review by SukantPal — APPROVED** [review]
Make sure to update the description when the value changes in the code.

**Comment by bigtimebuddy** [comment]
Thanks @ghsable :tada:

**Comment by ghsable** [comment]
Thanks 😆

**Pull request merged**